deployer/server.go

```diff
@@ -0,0 +1,406 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/flynn/flynn/Godeps/_workspace/src/github.com/bgentry/que-go"
+	"github.com/flynn/flynn/Godeps/_workspace/src/github.com/flynn/go-sql"
+	"github.com/flynn/flynn/Godeps/_workspace/src/github.com/flynn/pq"
+	"github.com/flynn/flynn/Godeps/_workspace/src/github.com/jackc/pgx"
+	"github.com/flynn/flynn/Godeps/_workspace/src/github.com/julienschmidt/httprouter"
+	"github.com/flynn/flynn/controller/client"
+	"github.com/flynn/flynn/deployer/strategies"
+	"github.com/flynn/flynn/deployer/types"
+	"github.com/flynn/flynn/discoverd/client"
+	"github.com/flynn/flynn/pkg/postgres"
+	"github.com/flynn/flynn/pkg/random"
+)
+
+var q *que.Client
+var db *postgres.DB
+var client *controller.Client
+
+var ErrNotFound = errors.New("deployer: resource not found")
+
+type deployID struct {
+	ID string
+}
+
+func main() {
+	var err error
+	client, err = controller.NewClient("", os.Getenv("CONTROLLER_AUTH_KEY"))
+	if err != nil {
+		log.Fatalln("Unable to connect to controller:", err)
```

Comment:
`controller.NewClient` doesn't actually connect to the controller, it only returns an error if it can't parse the URL or fails to connect to discoverd (see [here](https://github.com/flynn/flynn/blob/master/controller/client/client.go#L53-L80)), so this error should probably be `Unable to create controller client`